modules/fb_instant_articles_display/fb_instant_articles_display.info

```diff
@@ -2,6 +2,6 @@ name = Facebook Instant Articles Display
 description = Display module for Facebook Instant Articles.
 package = Facebook Instant Articles
 core = 7.x
-configure = admin/config/services/fb-instant-articles/display
 dependencies[] = ctools
 dependencies[] = fb_instant_articles
+scripts[] = js/admin.js
```

Comment:
This is already added by `$form['#attached']['js']` in `fb_instant_articles_display_layout_form()`. You'd only add this line in the info file if you want this JS to load on every page.